Made Changes to How to use SwiftCurrent with UIKit Guide (#134)
wwt/SwiftCurrent: .github/abstract/How to use SwiftCurrent with UIKit.md

> SwiftCurrent works well with UIKit. To work correctly, requires `FlowRepresentable`s to be `UIViewController`s and not `UIView`s. It comes with [support for Storyboards](using-storyboards.html) with `StoryboardLoadable` and [programmatic views](using-programmatic-views.html). 

Reviewer: Meaning got lost here I think???

I had to read this several times to understand the change. 

Verbose version:

To work correctly you done gotta make your `FlowRepresentable`s `UIViewController`s, if you make them `UIViews` things'll break, so don't do that.

Reviewer: ```suggestion
SwiftCurrent works well with UIKit. To work correctly, `FlowRepresentable`s need to be `UIViewController`s and not `UIView`s. It comes with [support for Storyboards](using-storyboards.html) with `StoryboardLoadable` and [programmatic views](using-programmatic-views.html). 
```

Reviewer 2: what about? 
```suggestion
SwiftCurrent works well with UIKit when `FlowRepresentable`s are `UIViewController`s. `UIView`s are not supported at this time. It comes with [support for Storyboards](using-storyboards.html) with `StoryboardLoadable` and [programmatic views](using-programmatic-views.html). 
```

Reviewer 3: +1

Author: @Richard-Gist Sounds good to me!
